### ISSUE TITLE
Error - Run test.lua: CuDNN: CUDNN_STATUS_NOT_INITIALIZED

### ISSUE THREAD

**Issue by chelicynly** [soumith/cudnn.torch#229]
When I run test.lua, I got lots of errors:

Running test on device: #1 : GeForce GTX 860M	
Testing torch.CudaHalfTensor	
Running 38 tests
 1/38 VolumetricMaxPooling_batch ........................................ [PASS]
 2/38 SpatialMaxPooling_single .......................................... [ERROR]
 3/38 SpatialCrossEntropyCriterion ...................................... [PASS]
 4/38 SpatialMaxPooling_batch ........................................... [PASS]
 5/38 SpatialAveragePooling_batch ....................................... [PASS]
 6/38 LogSoftMax_single ................................................. [ERROR]
 7/38 functional_bias2D ................................................. [ERROR]
 8/38 SpatialBatchNormalization ......................................... [ERROR]
 9/38 SpatialConvolution_forward_single ................................. [ERROR]
10/38 functional_convolution2d .......................................... [ERROR]
11/38 SpatialFullConvolution_backward_batch ............................. [ERROR]
12/38 TemporalConvolution_padding_batch ................................. [ERROR]
13/38 SpatialConvolution_backward_single ................................ [ERROR]
14/38 Tanh_single ....................................................... [ERROR]
15/38 ClippedReLU_single ................................................ [ERROR]
16/38 ReLU_single ....................................................... [ERROR]
17/38 Sigmoid_single .................................................... [ERROR]
18/38 VolumetricConvolution_forward_single .............................. [ERROR]
19/38 Tanh_batch ........................................................ [ERROR]
20/38 SoftMax_batch ..................................................... [ERROR]
21/38 SpatialCrossMapLRN_batch .......................................... [ERROR]
22/38 VolumetricMaxPooling_single ....................................... [ERROR]
23/38 VolumetricBatchNormalization ...................................... [ERROR]
24/38 ReLU_batch ........................................................ [ERROR]
25/38 SpatialConvolution_backward_batch ................................. [ERROR]
26/38 TemporalConvolution_batch ......................................... [ERROR]
27/38 SpatialFullConvolution_forward_batch .............................. [ERROR]
28/38 Sigmoid_batch ..................................................... [ERROR]
29/38 SpatialLogSoftMax ................................................. [ERROR]
30/38 TemporalConvolution_reduceBatchSize ............................... [ERROR]
31/38 LogSoftMax_batch .................................................. [ERROR]
32/38 BatchNormalization ................................................ [ERROR]
33/38 TemporalConvolution_single ........................................ [ERROR]
34/38 functional_maxpooling2d ........................................... [ERROR]
35/38 SpatialConvolution_forward_batch .................................. [ERROR]
36/38 SpatialAveragePooling_single ...................................... [ERROR]
37/38 VolumetricConvolution_backward_single ............................. [ERROR]
38/38 SoftMax_single .................................................... [ERROR]
Completed 7 asserts in 38 tests with 0 failures and 34 errors
--------------------------------------------------------------------------------
SpatialMaxPooling_single
 Function call failed
/home/yueru/torch/install/share/lua/5.1/cudnn/init.lua:75: Error in CuDNN: CUDNN_STATUS_NOT_INITIALIZED
stack traceback:
	[C]: in function 'error'
	/home/yueru/torch/install/share/lua/5.1/cudnn/init.lua:75: in function 'getHandle'
	/home/yueru/torch/install/share/lua/5.1/cudnn/init.lua:83: in function 'errcheck'
	/home/yueru/torch/install/share/lua/5.1/cudnn/Pooling.lua:32: in function 'resetPoolDescriptors'
	/home/yueru/torch/install/share/lua/5.1/cudnn/Pooling.lua:82: in function 'forward'
	test.lua:792: in function 'test'
	test.lua:809: in function <test.lua:767>
	[C]: in function 'xpcall'
	/home/yueru/torch/install/share/lua/5.1/torch/Tester.lua:477: in function '_pcall'
	/home/yueru/torch/install/share/lua/5.1/torch/Tester.lua:436: in function '_run'
	/home/yueru/torch/install/share/lua/5.1/torch/Tester.lua:355: in function 'run'
	test.lua:1525: in main chunk
	[C]: in function 'dofile'
	...ueru/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00406670

I use the cuda 7.5 and install cudnn v5. Could you help me about this?

**Comment by chelicynly** [comment]
I fixed it by install cudnn v5.1
